src/ttrssapi.cpp

```diff
@@ -152,7 +152,7 @@ json TtRssApi::run_op(const std::string& op,
 	}
 
 	std::string result = utils::retrieve_url(
-			url, cfg, auth_info, &req_data, cached_handle);
+			url, cfg, auth_info, &req_data, "POST", cached_handle);
```

Comment:
JFTR: [TT-RSS API reference](https://git.tt-rss.org/git/tt-rss/wiki/ApiReference) says that "Client parameters should be passed encoded using JSON in HTTP POST data (supported since version:1.5.3).", so this change shouldn't break anything. I don't have a TT-RSS instance, though, so I didn't really test it.